activesupport/lib/active_support/duration.rb

```diff
@@ -52,6 +52,13 @@ def ==(other)
       end
     end
 
+    # Returns the amount of seconds a duration covers as a string.
+    #
+    #   1.minute.to_s   # => "60"
+    #   1.hour.to_s     # => "3600"
+    #   1.day.to_s      # => "86400"
+    #   1.month.to_s    # => "2592000"
+    #   1.year.to_s     # => "31557600.0"
```

Comment:
in [`#to_i`](https://github.com/rails/rails/pull/20293/files#diff-273d4ceedb426a237b07cb7146a074dfL65) you can see that 1 year == 365.25 days. And so I think that `1.year.to_s` is float